portfolio/src/main/webapp/script.js

```diff
@@ -171,9 +171,14 @@ function createCommentElement(comment) {
   const timestampString = new Date(comment.timestamp).toString();
   timeElement.innerText = timestampString;
 
+  const scoreElement = document.createElement('div');
```

Comment:
A number called sentiment score is not very obvious to understand so you should consider changing this to some other form of display. Also it would be nice to show some description of what sentiment score is I hover the mouse over this (see how to do this with css - https://www.w3schools.com/css/css_tooltip.asp)